neuralop/training/patching.py

```diff
@@ -78,125 +97,169 @@ def _stitch(self, x):
         if self.skip_padding:
             return x
 
-        #Only 1D and 2D supported
-        assert x.ndim == 4, f'Only 2D patche supported but got input with {x.ndim} dims.'
-        
+        # Only 1D and 2D supported
+        assert x.ndim == 4, f"Only 2D patch supported but got input with {x.ndim} dims."
+
         if self.n_patches[0] <= 1 and self.n_patches[1] <= 1:
             return x
 
-        #Size with padding removed    
+        # Size with padding removed
         size = x.size()
 
         # if self.mode == "batch-wise":
-        B = size[0]//(self.n_patches[0]*self.n_patches[1])
-        W = size[3]*self.n_patches[1]
+        B = size[0] // (self.n_patches[0] * self.n_patches[1])
+        W = size[3] * self.n_patches[1]
```

Comment:
I really don't like the way `black` formats operations - it actually doesn't follow pep8..